README.md

```diff
@@ -1,25 +1,104 @@
-# RL_in_Search_space_exploration
+# RL for performance optimization
 
-## Running the script
-```bash
-sbatch run_rllib_slurm.sh
+The following sections represent the steps in order to successfully train the RL agent.
+
+## Generate the dataset
```

Comment:
We have added the code generator as a submodule. The generation is the same steps but with a few tweaks in the arguments